api/app/src/routes/medical/document.ts

```diff
@@ -27,8 +31,8 @@ router.get(
     const facilityId = getFromQueryOrFail("facilityId", req);
     const forceQuery = getFromQuery("force-query", req);
 
-    const documents = await getDocuments({ cxId, patientId });
-    const documentsDTO = documents.map(toDTO);
+    const documents = await getDocuments(patientId);
```

Comment:
Why aren't we calling our business logic/command here? Doing like this is tying our API to Medplum code, whereas the command would call `/external` and convert to our own shape/type.

If we're to leave as is, I think we should remove code we're not using anymore.

Reply:
Gotcha i was just under the impression that logic that lived within the commands was to interact with the models we have for the sql DB. 

If i understand correctly i should move just the getDocuments from medplum into a command and then reference that? After which the DTO still remains within the routes since thats where it lives

Reply:
As per standup conversation, we will stick with medplum for now